### ISSUE TITLE
Notification icon is not showing correctly in notification bar

### ISSUE THREAD

**Issue by junweimah** [caffeinalab/ti.goosh#44]
I am seeing this circle icon instead of my icon i have set in the payload

![img_20170301_103258](https://cloud.githubusercontent.com/assets/21700014/23443706/93b4a6fa-fe6b-11e6-8c7f-71d053b12432.jpg)

But when i pull down the notification bar, i can see the correct icon that I have set

![img_20170301_103306_01](https://cloud.githubusercontent.com/assets/21700014/23443724/afc9e0ee-fe6b-11e6-9d15-0e4a476cb8c8.jpg)

How do I change it so that I can see my icon even when the notification bar is not pulled down?

**Comment by junweimah** [comment]
Opppps

I found the source of the file, i just delete and paste new file to here

![image](https://cloud.githubusercontent.com/assets/21700014/23444044/9e025826-fe6d-11e6-90fa-0afd05652431.png)

you can close this, thanks

**Comment by junweimah** [comment]
I found the source of the file, just delete and paste new image file here

![image](https://cloud.githubusercontent.com/assets/21700014/23444066/b3e41756-fe6d-11e6-9dc9-2ad41f2f8f60.png)

you can close this, thanks

**Comment by Hrfreire** [comment]
I'm not able to set the icon. I've tried to set on ti.goosh/platform/android/res/drawable-*/notificationicon.png and on my project platform folder, but I continue seeing this white circle.

I have tested the same icon with local notification and it works. (White icon with transparent background)

I have tested on android 6.0.

I used the resolutions indicated for this module: 
22 × 22 area in 24 × 24 (mdpi)
33 × 33 area in 36 × 36 (hdpi)
44 × 44 area in 48 × 48 (xhdpi)
66 × 66 area in 72 × 72 (xxhdpi)
88 × 88 area in 96 × 96 (xxxhdpi)
